rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1192,6 +1192,127 @@ void saveSvpFundTransactionUnsignedHash_postLovell700AndResettingToNull_shouldSa
 
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("get fund transaction unsigned hash tests")
+    class GetSvpFundTransactionUnsignedHashTests {
+
+        private StorageAccessor storageAccessor;
+        private FederationStorageProvider federationStorageProvider;
+        private final ActivationConfig.ForBlock activations = ActivationConfigsForTest.all().forBlock(0L);
+
+        @BeforeEach
+        void setup() {
+            storageAccessor = new InMemoryStorage();
+            federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+        }
+
+        @Test
+        void getSvpFundTransactionUnsignedHash_preLovell700_shouldReturnEmpty() {
+
+            // Arrange
+
+            ActivationConfig.ForBlock arrowheadActivations = ActivationConfigsForTest.arrowhead631().forBlock(0L);
+
+            // Manually setting the value in storage to then assert that pre fork the method doesn't access the storage
+            Sha256Hash hash = BitcoinTestUtils.createHash(123_456_789);
+            storageAccessor.saveToRepository(SVP_FUND_TX_HASH_UNSIGNED.getKey(), BridgeSerializationUtils.serializeSha256Hash(hash));
+
+            // Act
+
+            Optional<Sha256Hash> svpFundTransactionUnsignedHash = federationStorageProvider.getSvpFundTransactionUnsignedHash(arrowheadActivations);
+
+            // Assert
+
+            assertEquals(Optional.empty(), svpFundTransactionUnsignedHash);
+
+        }
+
+        @Test
+        void getSvpFundTransactionUnsignedHash_whenThereIsNoProposedFederationSavedNorSet_shouldReturnEmpty() {
```

Comment:
you left the proposedFederation name jaja

Reply:
Updated.